src/Contract.php

```diff
@@ -94,10 +96,16 @@ public function sign(DocumentSignInterface $documentSign)
             $this->external_id = $document['id'];
 
             $redirectUrl = '';
-            if (Arr::has($config, 'return')) {
-                $redirectUrl = str_replace('{id}', $this->id, Arr::get($config, 'return', ''));
-            }
-            $documentSign->signer($this->user->email)->sign($document['id'], Arr::get($config, 'from_name'), Arr::get($config, 'from_email'), '', $redirectUrl);
+            Arr::has($config, 'return') && $redirectUrl = str_replace('{id}', $this->id, Arr::get($config, 'return', ''));
```

Comment:
this is not how we write this in php. Always start with the variabele because now it is not easy to read that a variabele is being set here.
$redirectUrl = Arr::has($config, 'return') ? '' : ''

Also you already checked for Arr::has so the Arr::get is not needed anymore you can just use $config['return'].